test/config_test.py

```diff
@@ -0,0 +1,113 @@
+import os
+import unittest
+
+from pathlib import Path
+
+import mongo_util
+from test_helper import create_temp_file
+
+
+DEFAULT_TEST_CONFIG_PATH = os.environ[mongo_util.TEST_CONFIG_ENV_PATH]
+
+
+class ConfigTest(unittest.TestCase):
+
+    def test_minimal_config(self):
+        data_string = """[HandleService2]\n
+        test.mongo.exe=mongod\n
+        test.temp.dir=temp_test_dir\n
+        auth-service-url=https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login\n
+        auth-url=https://ci.kbase.us/services/auth\n
+        shock-url=https://ci.kbase.us/services/shock-api\n
+        admin-token=998\n
+        test-token=998\n
+        admin-roles=HANDLE_ADMIN, KBASE_ADMIN\n
```

Comment:
```suggestion
        shock-url=https://ci.kbase.us/services/shock-api\n
        admin-token=998\n
        test-token=997\n
        admin-roles=HANDLE_ADMIN, KBASE_ADMIN\n
```
Ensure the token values don't get crossed

Reply:
👍

Reply:
:+1: